libs/cardano-protocol-tpraos/src/Cardano/Protocol/TPraos/API.hs

```diff
@@ -325,7 +326,7 @@ view
 --  application of the TICK rule at the target slot to the curernt ledger state.
 
 newtype FutureLedgerViewError era
-  = FutureLedgerViewError [PredicateFailure (EraRule "TICKF" era)]
+  = FutureLedgerViewError (NonEmpty (PredicateFailure (EraRule "TICKF" era)))
```

Comment:
Same here and for `ChainTransitionError`, this will require a major version bump and a changelog entry. I don't think any packages in this repo will be affected by this change, so I no lower bounds need to be changed

Reply:
`cardano-protocol-tpraos` is already at version `1.1.0.0` so I don't think it needs a version bump